<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">

<!-- Mirrored from mesquiteproject.wikispaces.com/Analyzing+Molecular+Data?f=print by HTTrack Website Copier/3.x [XR&CO'2014], Tue, 01 May 2018 13:27:42 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=UTF-8" /><!-- /Added by HTTrack -->
<head>
<meta http-equiv="Content-Type" content="text/html;charset=utf-8" />
<meta name="viewport" content="width=device-width, initial-scale=1.0" />
<link rel="search" href="http://mesquiteproject.wikispaces.com/space/opensearch" type="application/opensearchdescription+xml" title="Wikispaces Search (Mesquite Wiki)" />
<link rel="alternate" type="application/x-wiki" title="Edit This Page" href="http://mesquiteproject.wikispaces.com/page/edit/Analyzing+Molecular+Data" />
<link rel="alternate" title="Mesquite - All Changes" href="http://mesquiteproject.wikispaces.com/wiki/xmla?v=rss_2_0" type="application/rss+xml" />
<link rel="alternate" title="mesquiteproject : Analyzing Molecular Data - Discussion" href="http://mesquiteproject.wikispaces.com/message/xml/Analyzing+Molecular+Data?v=rss_2_0" type="application/rss+xml" />
<link rel="alternate" title="mesquiteproject : Analyzing Molecular Data - Edits" href="http://mesquiteproject.wikispaces.com/page/xml/Analyzing+Molecular+Data?v=rss_2_0" type="application/rss+xml" />
<link rel="microsummary" href="http://mesquiteproject.wikispaces.com/page/microsummary/Analyzing+Molecular+Data/edits"/>
<link rel="microsummary" href="http://mesquiteproject.wikispaces.com/page/microsummary/Analyzing+Molecular+Data/messages"/>
<meta name="robots" content="noindex" />
<script type="text/javascript">
  
if (typeof ws === 'undefined') ws = {};
ws.context = {
  instance: {
    id: '1',
    url: 'wikispaces.com',
    isMain: true,
    supportsGoogleDocs: false,
    isTesClassroom: false,
  },
  user: {
    id: '',
    username: '',
    status: '',
    locale: 'en',
    imageUrlPrefix: 'http://www.wikispaces.com/i/user_none_',
    isTeacher: false,
    canAdministerWiki: false,
  },
  wiki: {
    id: '21322146',
    name: 'mesquiteproject',
    text: 'Mesquite',
    module: 'E',
    isClassroom: false,
    isTesClassroom: false
    ,defaultSpaceId: 21506332
    ,isWwwWiki: false
  },
  project: {
    id: '21772858',
    name: ''
  },
  space: {
    id: '21506332',
    name: 'mesquiteproject',
    isDefaultSpace: true,
    isDefaultTemplate: false
  },

  environment_name: 'prod',

  path: {
    object: 'page',
    method: 'view',
    page: 'Analyzing Molecular Data',
    id: ''
  },

  settings: {
    main_domain: 'wikispaces.com',
    session_url: 'https://session.wikispaces.com/1/',
    asset_url_prefix: 'http://www.wikispaces.com',
    beta_features: false,
    isMobileSite: false,
    isUploadCapable: true,
    can_discuss: false,
    has_gdocs: false,
    isDialog: false,
    pwin: ''
  },
  clockSkewMillis: 1525181187146 - ((window.performance && window.performance.timing.responseStart > 0) ? window.performance.timing.responseStart : (new Date()).getTime())
}


// legacy javascript API variables, use the above instead
var wikispaces_isUserLoggedIn = !!ws.context.user.id;
var wikispaces_isMain = ws.context.instance.isMain;
var wikispaces_username = ws.context.user.username;
var wikispaces_spaceName = ws.context.space.name;
var wikispaces_projectName = ws.context.project.name;
var wikispaces_wikiName = ws.context.wiki.name;
var wikispaces_wikiText = ws.context.wiki.text;
var wikispaces_isDefaultSpace = ws.context.space.isDefaultSpace;
var wikispaces_method = ws.context.path.method;
var wikispaces_object = ws.context.path.object;
var wikispaces_page = ws.context.path.page;
var wikispaces_id = ws.context.path.id;
var wikispaces_domain = ws.context.instance.url;
var wikispaces_isUserOrganizer = false;
var wikispaces_main_domain = ws.context.settings.asset_url_prefix;
var wikispaces_asset_url_prefix = ws.context.settings.asset_url_prefix;
var wikispaces_beta_features = ws.context.settings.beta_features;
var wikispaces_isMobileSite = ws.context.settings.isMobileSite;
var wikispaces_isUploadCapable = ws.context.settings.isUploadCapable;
var wikispaces_can_discuss = ws.context.settings.can_discuss;
var wikispaces_has_gdocs = ws.context.settings.has_gdocs;

</script>  <title>Mesquite - Analyzing Molecular Data</title>
  <meta charset="utf-8" />
  <link rel="apple-touch-icon" sizes="144x144" href="http://www.wikispaces.com/_/z6371z4x/i/logo-144.png" />
  <link rel="apple-touch-icon" sizes="114x114" href="http://www.wikispaces.com/_/j5j77xz3/i/logo-114.png" />
  <link rel="apple-touch-icon" sizes="72x72" href="http://www.wikispaces.com/_/vq8j5432/i/logo-72.png" />
  <link rel="apple-touch-icon" href="http://www.wikispaces.com/_/38515190/i/logo-57.png" />
  <link rel="shortcut icon" href="http://www.wikispaces.com/_/55v9k02z/i/logo-16.png" />
  <style type="text/css">
    #foot { margin-top: 5px; padding: 5px 0; font-family: arial,helvetica,sans-serif; font-size: 90%; line-height: 150%; clear: both; }
    h1.pageTitle { font-family: arial,helvetica,sans-serif; }
  </style>
<link rel="stylesheet" href="http://www.wikispaces.com/stylesheet/viewi/217420326D.gz.css" type="text/css" />
<link rel="stylesheet" href="http://www.wikispaces.com/s/gz/9jvvjv40-internal-bootstrap.css" type="text/css" />
<link rel="stylesheet" href="http://www.wikispaces.com/s/gz/vz033906-internal.css" type="text/css" />
<style type="text/css">
          .WikiControls { display: none; }
          </style><script src="http://www.wikispaces.com/s/gz/52vxz59q-top.min.js" type="text/javascript"></script>

<script type="text/javascript">
  var _gaq = _gaq || [];

  var wsGoogleAccount = 'UA-80504-1';
  _gaq.push(['_setAccount', wsGoogleAccount]);
  _gaq.push(['_gat._anonymizeIp']);
  _gaq.push(['_setDomainName', 'wikispaces.com']);
  _gaq.push(['_setAllowLinker', true]);
  _gaq.push(['_setCustomVar', 2, 'Authenticated', 'Guest', 2]);
  _gaq.push(['_trackPageview']);
</script></head>
<body data-enhance="false">
<table cellpadding="0" cellspacing="0" border="0" width="100%">
<tr>
  <td valign="top">
    <div style="padding: 10px;">
<h1 class="pageTitle">Analyzing Molecular Data</h1>
<div class="contentBox"><div class="innerContentBox" id="WikiContent">

<script type="text/template" id="comment-footer-template">

<div class="commentFooter modal-footer">
  <div class="pull-left">
    <button type="button" class="btn btn-default btn-sm previous"><i class="fa fa-arrow-left"></i></button>
    <button type="button" class="btn btn-default btn-sm next"><i class="fa fa-arrow-right"></i></button>
  </div>
  <% if (ws.context.user.canAdministerWiki && !ws.comments.Manager.readOnly) { %>
  <div class="pull-right">
    <button type="button" class="btn btn-default btn-sm remove">Remove</button>
  </div>
  <% } %>
</div>

</script>

<script type="text/template" id="comment-content-template">
<div class="ws-box commentView">
  <div class="comment-close">
    <button type="button" class="closeme close" aria-hidden="true"><i class="fa fa-times"></i></button>
  </div>
  <div class="commentHeader">
    <p class="ws-stream-meta command-context">
      <a class="userLink userLinkPicture" href="<%= userCreated.url %>"><img class="userPicture" src="<%= userCreated.imageUrl %>" width="16" height="16"></a>
      <a href="<%= userCreated.url %>"><%= userCreated.username %></a>
      <% if (smartDate == '__live__') { %>
      <abbr class="timeago" title="<%= (new Date(parseInt(dateCreated, 10) * 1000)).toISOString() %>">
        <%= dateCreated %>
      </abbr>
      <% } else { %>
        <%- smartDate %>
      <% } %>
      <% if (ws.context.user.canAdministerWiki && ws.comments.Manager.readOnly) { %>
        <i class="fa fa-trash-o contextual-command delete ws-tip" title="Delete Comment"></i>
      <% } %>
    </p>
  </div>

  <div class="commentBody">
    <div class="commentContent">
      <%= description %>
    </div>

      <% for(i = 0; i < replyPages.length; i++) { %>
        <% for(var j = 0; j < replyPages[i].digests.length; j++) { %>
      <div class="commentReply" data-id="<%= replyPages[i].digests[j].id %>">
        <p class="ws-stream-meta command-context">
          <a class="userLink userLinkPicture" href="<%= replyPages[i].digests[j].userCreated.url %>"><img class="userPicture" src="<%= replyPages[i].digests[j].userCreated.imageUrl %>" width="16" height="16"></a>
          <a href="<%= replyPages[i].digests[j].userCreated.url %>"><%= replyPages[i].digests[j].userCreated.username %></a>
          <% if (smartDate == '__live__') { %>
          <abbr class="timeago" title="<%= (new Date(parseInt(replyPages[i].digests[j].dateCreated, 10) * 1000)).toISOString() %>">
            <%= new Date(parseInt(replyPages[i].digests[j].dateCreated, 10) * 1000).toLocaleString() %>
          </abbr>
          <% } else { %>
            <%- smartDate %>
          <% } %>
          <% if (ws.context.user.canAdministerWiki) { %>
          <i class="fa fa-trash-o contextual-command delete-reply ws-tip" title="Delete Reply"></i>
          <% } %>
          </p>
        <div class="commentContent"><%= replyPages[i].digests[j].body %></div>
      </div>
        <% } %>
      <% } %>

      <form class="reply commentReply">
        <div class="form-group">
          <textarea name="reply" placeholder="Reply" class="form-control" /></textarea>
        </div>
        <div class="btn-toolbar">
          <button type="submit" class="btn btn-primary btn-sm" name="go" value="1">Post</button>

          <% if (ws.context.user.id) { %>
            <% if (monitored) { %>
            <button type="button" class="btn btn-default btn-sm pull-right toggle-monitor ws-tip active" title="You are monitoring replies."><i class="fa fa-eye"></i></button>
            <% } else { %>
            <button type="button" class="btn btn-default btn-sm pull-right toggle-monitor ws-tip" title="You are not monitoring replies."><i class="fa fa-eye"></i></button>
            <% } %>
          <% } %>
          <% if (ws.context.user.canAdministerWiki) { %>
            <% if (locked) { %>
            <button type="button" class="btn btn-default btn-sm pull-right toggle-lock ws-tip active" title="Replies are not allowed."><i class="fa fa-lock"></i></button>
            <% } else { %>
            <button type="button" class="btn btn-default btn-sm pull-right toggle-lock ws-tip" title="Replies are allowed."><i class="fa fa-lock"></i></button>
            <% } %>
          <% } %>

          <i class="fa fa-spinner fa-spin spinner pull-right hidden"></i>
        </div>
      </form>
   </div>

   
<div class="commentFooter modal-footer">
  <div class="pull-left">
    <button type="button" class="btn btn-default btn-sm previous"><i class="fa fa-arrow-left"></i></button>
    <button type="button" class="btn btn-default btn-sm next"><i class="fa fa-arrow-right"></i></button>
  </div>
  <% if (ws.context.user.canAdministerWiki && !ws.comments.Manager.readOnly) { %>
  <div class="pull-right">
    <button type="button" class="btn btn-default btn-sm remove">Remove</button>
  </div>
  <% } %>
</div>

</div>
</script>

<script type="text/template" id="comment-composer-template">
<div class="ws-box commentView">
  <div class="commentComposer">
    <div class="comment-close">
      &nbsp;<button type="button" class="close closeme" aria-hidden="true"><i class="fa fa-times"></i></button>
    </div>
    <div class="commentHeader">
      <p class="ws-stream-meta command-context">
        <img class="userPicture" src="<%= userCreated.imageUrl %>" width="16" height="16">
        <a href="#" onclick="return false;"><%= userCreated.username %></a>
      </p>
    </div>
    <div class="commentBody">
      <textarea name="comment" class="compose form-control" placeholder="Comment"/></textarea><br/>
      <div class="btn-toolbar">
        <button type="button" class="btn btn-default btn-sm monitored active pull-right" style="margin-left: 6px" onclick="jQuery(this).toggleClass('active')"><i class="fa fa-eye"></i></button>
        <button type="button" class="btn btn-default btn-sm locked btn-toggle pull-right" onclick="jQuery(this).toggleClass('active')"><i class="fa fa-lock"></i></button>
        <button type="button" class="btn btn-primary btn-sm closeme">Save</button>
      </div>
    </div>
  </div>

  
<div class="commentFooter modal-footer">
  <div class="pull-left">
    <button type="button" class="btn btn-default btn-sm previous"><i class="fa fa-arrow-left"></i></button>
    <button type="button" class="btn btn-default btn-sm next"><i class="fa fa-arrow-right"></i></button>
  </div>
  <% if (ws.context.user.canAdministerWiki && !ws.comments.Manager.readOnly) { %>
  <div class="pull-right">
    <button type="button" class="btn btn-default btn-sm remove">Remove</button>
  </div>
  <% } %>
</div>

</div>
</script>

<script type="text/template" id="comment-proxy-template">
<div class="comment-proxy">
  <span class="commentUser">
  <% if (userCreated) { %>
    <img class="userPicture" src="<%= userCreated.imageUrl %>" width="16" height="16">
  <% } else { %>
    <img class="userPicture" src="<%= ws.context.user.imageUrlPrefix %>sm.jpg" width="16" height="16">
  <% } %>
  </span>
  <span class="commentReplies"><%= 1 + _(replyPages).reduce(function(memo, page){ return memo + page.digests.length; }, 0) %></span>
</div>
</script>







<div class="commentContainer">
  <div class="commentSubject">
  <div class="wiki wikiPage" id="content_view">
  <!-- ws:start:WikiTextHeadingRule:0:&lt;h1&gt; --><h1 id="toc0"><a name="Analyzing Molecular Data"></a><!-- ws:end:WikiTextHeadingRule:0 -->Analyzing Molecular Data</h1>
 Molecular data (DNA or protein sequences) can be simulated and analyzed in various ways in Mesquite. Most of the features discussed elsewhere concerning editing and analysis of general categorical data also apply to molecular data; here we focus on features specifically designed for sequence data.<br />
<!-- ws:start:WikiTextHeadingRule:2:&lt;h4&gt; --><h4 id="toc1"><a name="Analyzing Molecular Data---Contents"></a><!-- ws:end:WikiTextHeadingRule:2 -->Contents</h4>
 <ul><li><a class="wiki_link" href="Analyzing%20Molecular%20Data.html#genbank">BLASTing sequences</a></li><li><a class="wiki_link" href="Analyzing%20Molecular%20Data.html#simDNA">Simulating DNA sequence evolution</a></li><li><a class="wiki_link" href="Analyzing%20Molecular%20Data.html#statsDNA">Statistics for DNA sequences</a></li><li><a class="wiki_link" href="Analyzing%20Molecular%20Data.html#distances">DNA Distances</a></li><li><a class="wiki_link" href="Analyzing%20Molecular%20Data.html#statsProtein">Statistics for Protein sequences</a></li><li><a class="wiki_link" href="Analyzing%20Molecular%20Data.html#tertiary">Visualizing tertiary structure</a></li><li><a class="wiki_link" href="Analyzing%20Molecular%20Data.html#popGen">Sequence Data within populations</a></li><li><a class="wiki_link" href="Analyzing%20Molecular%20Data.html#rec">Reconstructing ancestral states</a></li></ul><br />
<!-- ws:start:WikiTextHeadingRule:4:&lt;h2&gt; --><h2 id="toc2"><a name="Analyzing Molecular Data-BLASTing sequences"></a><!-- ws:end:WikiTextHeadingRule:4 -->BLASTing sequences</h2>
 The following features allow one to BLAST sequences against GenBank or a local BLASTable database.<br />
<!-- ws:start:WikiTextHeadingRule:6:&lt;h3&gt; --><h3 id="toc3"><a name="Analyzing Molecular Data-BLASTing sequences-BLAST in Web Browser"></a><!-- ws:end:WikiTextHeadingRule:6 -->BLAST in Web Browser</h3>
 Select a sequence or portion thereof in the data matrix. Choose Matrix&gt;Search&gt;BLAST in Web Browser, and Mesquite will send a BLAST request to GenBank to search for matching sequences. Your default web browser should open and take you to NCBI's BLAST page.<br />
<!-- ws:start:WikiTextHeadingRule:8:&lt;h3&gt; --><h3 id="toc4"><a name="Analyzing Molecular Data-BLASTing sequences-Top BLAST Matches"></a><!-- ws:end:WikiTextHeadingRule:8 --><!-- ws:start:WikiTextAnchorRule:26:&lt;img src=&quot;/i/anchor.gif&quot; class=&quot;WikiAnchor&quot; alt=&quot;Anchor&quot; id=&quot;wikitext@@anchor@@TopBlast&quot; title=&quot;Anchor: TopBlast&quot;/&gt; --><a name="TopBlast"></a><!-- ws:end:WikiTextAnchorRule:26 -->Top BLAST Matches</h3>
 This tool will search a database using BLAST, and return information about the top hits, and optionally import them into the current matrix. You will be given the choice to either BLAST NCBI's database, or BLAST a local database that you have created. To use this tool, select a sequence or portion thereof in the data matrix. Choose Matrix&gt;Search&gt;Top BLAST Matches. You will be given the choice between BLASTing the NCBI GenBank Server, or BLASTing a local database, on your computer.<br />
If you choose to BLAST GenBank directly, you will be asked to choose among various options, including:<br />
<ul><li>maximum number of matches to be sought by Mesquite, and the eValue cutoff for acceptable hits</li><li>whether the report of results should be saved to a text file</li><li>whether details of the taxonomic lineage of the sequences should be reported</li><li>whether to import the top matches into the current matrix</li><li>the maximum time in seconds to wait for the BLAST search to complete.</li></ul><br />
If you wish instead to BLAST a local database, perhaps containing sequences you generated, then you will first need to create a BLASTable database. You can do this by following the instructions on this <a class="wiki_link_ext" href="http://www.ncbi.nlm.nih.gov/guide/howto/run-blast-local" rel="nofollow">NCBI BLAST page</a>. After you install the NCBI tools, you can turn a FASTA file containing your sequences using the following instructions. Let's imagine that your FASTA file is called &quot;FASTAfileName&quot;, and that you wish to create a BLASTable nucleotide database called &quot;BLASTableDatabaseName&quot;. Then use the following command in your computer's terminal shell:<br />
<br />
<strong>makeblastdb -in FASTAfileName -out BLASTableDatabaseName -dbtype nucl</strong><br />
<br />
This will create three files, BLASTableDatabaseName.nin, BLASTableDatabaseName.nsq, and BLASTableDatabaseName.nhr. Move these to whatever folder is required for NCBI's BLAST tools to function (e.g., on the macOS, this is a &quot;db&quot; folder in a &quot;blast&quot; folder in your home directory). You can then BLAST the BLASTableDatabaseName file directly from within Mesquite. Within Mesquite, after you choose to BLAST a local database, you will first be asked to specify some details about the local database before you choose the BLAST options in the &quot;Local BLAST Options&quot; dialog box. In the &quot;Database to search&quot;, put the name of the database (e.g., &quot;BLASTableDatabaseName&quot;). You needn't enter any other options in this dialog box for databases you have created yourself. However, if you are going to do a blastX to a local protein database that you downloaded from GenBank, you will need to check &quot;Use ID in Definition&quot; box.<br />
<br />
<!-- ws:start:WikiTextHeadingRule:10:&lt;h3&gt; --><h3 id="toc5"><a name="Analyzing Molecular Data-BLASTing sequences-Fetch &amp; Add GenBank Sequences"></a><!-- ws:end:WikiTextHeadingRule:10 -->Fetch &amp; Add GenBank Sequences</h3>
 This option, in Matrix&gt;Utilities&gt; Fetch &amp; Add GenBank Sequences, allows one to enter a comma-delimited list of GenBank accession numbers, and Mesquite will acquire these from GenBank and import them into the current matrix.<br />
<br />
<!-- ws:start:WikiTextHeadingRule:12:&lt;h2&gt; --><h2 id="toc6"><a name="Analyzing Molecular Data-Simulating DNA sequence evolution"></a><!-- ws:end:WikiTextHeadingRule:12 --><!-- ws:start:WikiTextAnchorRule:27:&lt;img src=&quot;/i/anchor.gif&quot; class=&quot;WikiAnchor&quot; alt=&quot;Anchor&quot; id=&quot;wikitext@@anchor@@simDNA&quot; title=&quot;Anchor: simDNA&quot;/&gt; --><a name="simDNA"></a><!-- ws:end:WikiTextAnchorRule:27 -->Simulating DNA sequence evolution</h2>
 DNA sequence evolution can be simulated to build statistical tests, for instance via parametric bootstrapping. See the page on <a class="wiki_link" href="Simulating%20DNA%20Evolution.html">simulating DNA sequences</a>.<br />
<!-- ws:start:WikiTextHeadingRule:14:&lt;h2&gt; --><h2 id="toc7"><a name="Analyzing Molecular Data-Statistics for DNA sequences"></a><!-- ws:end:WikiTextHeadingRule:14 --><!-- ws:start:WikiTextAnchorRule:28:&lt;img src=&quot;/i/anchor.gif&quot; class=&quot;WikiAnchor&quot; alt=&quot;Anchor&quot; id=&quot;wikitext@@anchor@@statsDNA&quot; title=&quot;Anchor: statsDNA&quot;/&gt; --><a name="statsDNA"></a><!-- ws:end:WikiTextAnchorRule:28 -->Statistics for DNA sequences</h2>
 Calculations for categorical characters in general can be applied to DNA sequences. For example, <a class="wiki_link" href="Ancestral%20States.html#parsimony">parsimony calculations</a> can be made for DNA sequences, as can basic descriptive statistics such as the percent of a sequence or character that is missing data or gaps. In addition, there are several modules specifically designed for DNA data, illustrated by examples in Mesquite_Folder/examples/Molecular. These calculate compositional bias:<br />
<ul><li><!-- ws:start:WikiTextAnchorRule:29:&lt;img src=&quot;/i/anchor.gif&quot; class=&quot;WikiAnchor&quot; alt=&quot;Anchor&quot; id=&quot;wikitext@@anchor@@acgtBias&quot; title=&quot;Anchor: acgtBias&quot;/&gt; --><a name="acgtBias"></a><!-- ws:end:WikiTextAnchorRule:29 -->ACGT Compositional Bias — This module supplies the compositional bias of taxa, measured over the taxon's sequence. The bias is treated as a continuous character, and thus can be used wherever characters are used, as for instance in the reconstruction of the evolution of compositional bias as shown in the image below. It can return either the proportion G+C, or separately A, C, G, and T proportions.</li></ul><br />
<!-- ws:start:WikiTextLocalImageRule:124:&lt;div style=&quot;text-align: center&quot;&gt;&lt;img src=&quot;/file/view/gcBiasTaxa.jpg/519400770/gcBiasTaxa.jpg&quot; alt=&quot;&quot; title=&quot;&quot; /&gt;&lt;/div&gt; --><div style="text-align: center"><img src="http://mesquiteproject.wikispaces.com/file/view/gcBiasTaxa.jpg/519400770/gcBiasTaxa.jpg" alt="gcBiasTaxa.jpg" title="gcBiasTaxa.jpg" /></div><!-- ws:end:WikiTextLocalImageRule:124 --><br />
<br />
<ul><li><!-- ws:start:WikiTextAnchorRule:30:&lt;img src=&quot;/i/anchor.gif&quot; class=&quot;WikiAnchor&quot; alt=&quot;Anchor&quot; id=&quot;wikitext@@anchor@@charBias&quot; title=&quot;Anchor: charBias&quot;/&gt; --><a name="charBias"></a><!-- ws:end:WikiTextAnchorRule:30 -->Character Compositional Bias — This module supplies the compositional bias for characters. It calculates the percent of taxa with particular nucleotides (GC bias, or individual frequency of A, C, G or T) for a character. The image below shows a moving window analysis of compositional bias along a sequence; the instructions for generating the chart are given <a class="wiki_link_ext" href="http://mesquiteproject.org/Mesquite_Folder/docs/mesquite/charts.html#gcbias" rel="nofollow">here</a>.</li></ul><!-- ws:start:WikiTextLocalImageRule:125:&lt;div style=&quot;text-align: center&quot;&gt;&lt;img src=&quot;/file/view/gcSequence.jpg/518884706/gcSequence.jpg&quot; alt=&quot;&quot; title=&quot;&quot; /&gt;&lt;/div&gt; --><div style="text-align: center"><img src="http://mesquiteproject.wikispaces.com/file/view/gcSequence.jpg/518884706/gcSequence.jpg" alt="gcSequence.jpg" title="gcSequence.jpg" /></div><!-- ws:end:WikiTextLocalImageRule:125 --><br />
<ul><li><!-- ws:start:WikiTextAnchorRule:31:&lt;img src=&quot;/i/anchor.gif&quot; class=&quot;WikiAnchor&quot; alt=&quot;Anchor&quot; id=&quot;wikitext@@anchor@@colorGCBias&quot; title=&quot;Anchor: colorGCBias&quot;/&gt; --><a name="colorGCBias"></a><!-- ws:end:WikiTextAnchorRule:31 -->GC bias coloring of matrices — The cells of the Character Matrix Editor may be colored according to a moving window of GC bias along the sequence, as shown below, by selecting Display<br />
&gt;Color Cells&gt;Color By Cell Value, then once shown the colors can be smoothed by a moving window analysis by selecting Display&gt;Moving Window (for colors).</li></ul><br />
<!-- ws:start:WikiTextLocalImageRule:126:&lt;div style=&quot;text-align: center&quot;&gt;&lt;img src=&quot;/file/view/gcBiasColorMatrix.jpg/519400758/gcBiasColorMatrix.jpg&quot; alt=&quot;&quot; title=&quot;&quot; /&gt;&lt;/div&gt; --><div style="text-align: center"><img src="http://mesquiteproject.wikispaces.com/file/view/gcBiasColorMatrix.jpg/519400758/gcBiasColorMatrix.jpg" alt="gcBiasColorMatrix.jpg" title="gcBiasColorMatrix.jpg" /></div><!-- ws:end:WikiTextLocalImageRule:126 --><br />
<br />
<!-- ws:start:WikiTextHeadingRule:16:&lt;h2&gt; --><h2 id="toc8"><a name="Analyzing Molecular Data-DNA Distances"></a><!-- ws:end:WikiTextHeadingRule:16 --><!-- ws:start:WikiTextAnchorRule:32:&lt;img src=&quot;/i/anchor.gif&quot; class=&quot;WikiAnchor&quot; alt=&quot;Anchor&quot; id=&quot;wikitext@@anchor@@distances&quot; title=&quot;Anchor: distances&quot;/&gt; --><a name="distances"></a><!-- ws:end:WikiTextAnchorRule:32 -->DNA Distances</h2>
 Mesquite supports several distances for DNA data:<br />
<br />
<ul><li>Uncorrected (P) distances</li><li>Jukes-Cantor</li><li>Kimura 2-parameter</li><li>F81 (Felsenstein, 1981)</li><li>F84 (Felsenstein, 1984)</li></ul>There are several options available (in the Distance Parameters submenu) for dealing with ambiguous bases and gaps:<br />
<br />
<br />
<ul><li>Count Sites with Gap in Pair: If this is chosen, then when the distance between two sequences is calculated, then sites that contain a gap in one of the sequences but not the other will be by default included in the calculation. If this option is unchecked, then any site at which either sequence is a gap will be excluded.</li><li>Estimate Ambiguity Differences: If this is chosen, then sites in which one member of the pair has an ambiguity have their contribution to the distance estimated based upon the base patterns at non-ambiguous sites.</li><li>Base Frequence on Entire Matrix: If this is chosen, then those distances that use as parameters the frequencies of bases (A, C, G, and T) will calculate those frequencies based on all sequences; if it is not chosen, then the frequencies are calculated based upon just the two sequences being compared at any one time.</li></ul><br />
<!-- ws:start:WikiTextHeadingRule:18:&lt;h2&gt; --><h2 id="toc9"><a name="Analyzing Molecular Data-Statistics for Protein Data"></a><!-- ws:end:WikiTextHeadingRule:18 --><!-- ws:start:WikiTextAnchorRule:33:&lt;img src=&quot;/i/anchor.gif&quot; class=&quot;WikiAnchor&quot; alt=&quot;Anchor&quot; id=&quot;wikitext@@anchor@@statsProtein&quot; title=&quot;Anchor: statsProtein&quot;/&gt; --><a name="statsProtein"></a><!-- ws:end:WikiTextAnchorRule:33 -->Statistics for Protein Data</h2>
 <br />
<ul><li><!-- ws:start:WikiTextAnchorRule:34:&lt;img src=&quot;/i/anchor.gif&quot; class=&quot;WikiAnchor&quot; alt=&quot;Anchor&quot; id=&quot;wikitext@@anchor@@siteHydrophobicity&quot; title=&quot;Anchor: siteHydrophobicity&quot;/&gt; --><a name="siteHydrophobicity"></a><!-- ws:end:WikiTextAnchorRule:34 -->Site hydrophobicity — This module supplies the average amino acid hydrophobicity, averaged across taxa, for each site. It can be used in charts, for instance to see the relationship between a phylogenetic statistic for the site (character) and it average hydrophobicity. This <a class="wiki_link" href="Charts.html#varhydrophob">chart</a>, for example, shows parsimony character steps as a function of hydrophobicity:</li></ul><br />
<!-- ws:start:WikiTextLocalImageRule:127:&lt;div style=&quot;text-align: center&quot;&gt;&lt;img src=&quot;/file/view/stepsVsHydro.jpg/518884800/stepsVsHydro.jpg&quot; alt=&quot;&quot; title=&quot;&quot; /&gt;&lt;/div&gt; --><div style="text-align: center"><img src="http://mesquiteproject.wikispaces.com/file/view/stepsVsHydro.jpg/518884800/stepsVsHydro.jpg" alt="stepsVsHydro.jpg" title="stepsVsHydro.jpg" /></div><!-- ws:end:WikiTextLocalImageRule:127 --><br />
<br />
<br />
<ul><li><!-- ws:start:WikiTextAnchorRule:35:&lt;img src=&quot;/i/anchor.gif&quot; class=&quot;WikiAnchor&quot; alt=&quot;Anchor&quot; id=&quot;wikitext@@anchor@@aaHydrophobicity&quot; title=&quot;Anchor: aaHydrophobicity&quot;/&gt; --><a name="aaHydrophobicity"></a><!-- ws:end:WikiTextAnchorRule:35 -->Amino Acid hydrophobicity — The cells of the Character Matrix Editor may be colored according to a moving window of hydrophobicity along the sequence, as shown below, by selecting Display&gt;Color Cells&gt;Color By Cell Value, then once shown the colors can be smoothed by a moving window analysis by selecting Display&gt;Moving Window (for colors).</li></ul><br />
<!-- ws:start:WikiTextLocalImageRule:128:&lt;div style=&quot;text-align: center&quot;&gt;&lt;img src=&quot;/file/view/aahydroColorMatrix.jpg/519400470/aahydroColorMatrix.jpg&quot; alt=&quot;&quot; title=&quot;&quot; /&gt;&lt;/div&gt; --><div style="text-align: center"><img src="http://mesquiteproject.wikispaces.com/file/view/aahydroColorMatrix.jpg/519400470/aahydroColorMatrix.jpg" alt="aahydroColorMatrix.jpg" title="aahydroColorMatrix.jpg" /></div><!-- ws:end:WikiTextLocalImageRule:128 --><br />
<!-- ws:start:WikiTextHeadingRule:20:&lt;h2&gt; --><h2 id="toc10"><a name="Analyzing Molecular Data-Visualizing tertiary structure"></a><!-- ws:end:WikiTextHeadingRule:20 --><!-- ws:start:WikiTextAnchorRule:36:&lt;img src=&quot;/i/anchor.gif&quot; class=&quot;WikiAnchor&quot; alt=&quot;Anchor&quot; id=&quot;wikitext@@anchor@@tertiary&quot; title=&quot;Anchor: tertiary&quot;/&gt; --><a name="tertiary"></a><!-- ws:end:WikiTextAnchorRule:36 -->Visualizing tertiary structure</h2>
 Although there are not yet dedicated windows for visualizing phylogenetic statistics in the context of molecular structure, features have been added to the Scattergram chart to allow it to be adapted for this purpose. For instance, in this image cytochrome B is shown, with the amino acids colored according to a simple phylogenetic statistic: the number of parsimony steps on a phylogeny. The colors are smoothed by a moving window, and show that several coils of the molecule, a few at the left and one deep at the right, evolve more rapidly than others. This example is illustrated in the data file at Mesquite_Folder/examples/Molecular/06-cytochromeB.nex<br />
<!-- ws:start:WikiTextLocalImageRule:129:&lt;div style=&quot;text-align: center&quot;&gt;&lt;img src=&quot;/file/view/cytStructure.jpg/519400708/cytStructure.jpg&quot; alt=&quot;&quot; title=&quot;&quot; /&gt;&lt;/div&gt; --><div style="text-align: center"><img src="http://mesquiteproject.wikispaces.com/file/view/cytStructure.jpg/519400708/cytStructure.jpg" alt="cytStructure.jpg" title="cytStructure.jpg" /></div><!-- ws:end:WikiTextLocalImageRule:129 --><br />
To build such a chart, begin with a file with a matrix of protein sequences. The procedure is also described in the example files 08-cytochromeBlinked.nex and 09-cytochromeBscatter.nex.<br />
<br />
<ul><li>Select New Linked Matrix from the Characters menu. When a matrix is made to be linked to a second matrix, the two matrices are constrained to have the same number of characters.</li><li>Indicate that you want the linked matrix to be a Continuous matrix, and link it to your protein matrix. Then, turn it into a three dimensional matrix (Taxa X Characters X Coordinates [x, y and z]) by using Add Item and Rename Item in the Utilities submenu of the Matrix menu of the Character Matrix Editor. The x,y,z coordinates could be added for all taxa if known, but otherwise only one taxon needs to be filled out (because we will use the average x,y,z coordinates for the amino acids).</li><li>Once the linked matrix of xyz amino acid positions is entered, select Analysis&gt;New Scattergram for&gt; Characters. Indicate you want the scattergram to be for Stored Characters. (If you have &quot;Use Stored Characters/Matrices by Default&quot; turned on in the Defaults submenu if the File menu, Mesquite won't ask you and will simply use Stored Characters.) Indicate Same value for the two axes. In the dialog box &quot;Values for axes&quot;, choose Mean Value of Character (Linked Matrix). In response to &quot;Use characters from which matrix? (for Character Source)&quot; choose the protein sequence matrix as the matrix to be used. This will plot the sites (amino acids, characters) in their correct places, but as a series of round spots.</li><li>To change the appearance of the plot, select Join the Dots in the Special Effects submenu of the Scattergram menu. Then select Thick Joints, deselect Show Dots, deselect Join First to Last, and set the marker size larger (e.g., 8). This will result in a plot as shown above, but without the colors.</li><li>Next, choose Color by Third Value from the Colors menu and choose the value by which to color the amino acids. For parsimony steps, for instance, choose under Character Value with current tree.</li><li>Finally, to use a moving window to smooth the colors, select Moving Window for Colors from the Colors menu and indicate the window size (e.g., 5).</li></ul><br />
<!-- ws:start:WikiTextHeadingRule:22:&lt;h2&gt; --><h2 id="toc11"><a name="Analyzing Molecular Data-Sequence data within populations"></a><!-- ws:end:WikiTextHeadingRule:22 --><!-- ws:start:WikiTextAnchorRule:37:&lt;img src=&quot;/i/anchor.gif&quot; class=&quot;WikiAnchor&quot; alt=&quot;Anchor&quot; id=&quot;wikitext@@anchor@@popGen&quot; title=&quot;Anchor: popGen&quot;/&gt; --><a name="popGen"></a><!-- ws:end:WikiTextAnchorRule:37 -->Sequence data within populations</h2>
 See the page on <a class="wiki_link" href="Gene%20Trees.html">population genetics</a>.<br />
<!-- ws:start:WikiTextHeadingRule:24:&lt;h2&gt; --><h2 id="toc12"><a name="Analyzing Molecular Data-Reconstructing ancestral states"></a><!-- ws:end:WikiTextHeadingRule:24 --><!-- ws:start:WikiTextAnchorRule:38:&lt;img src=&quot;/i/anchor.gif&quot; class=&quot;WikiAnchor&quot; alt=&quot;Anchor&quot; id=&quot;wikitext@@anchor@@rec&quot; title=&quot;Anchor: rec&quot;/&gt; --><a name="rec"></a><!-- ws:end:WikiTextAnchorRule:38 -->Reconstructing ancestral states</h2>
 Ancestral states of molecular characters can be reconstructed as described in the page on <a class="wiki_link" href="Ancestral%20States.html">reconstructing ancestral states</a>. Likelihood methods are not yet available for molecular characters.
  </div>


  </div>
  <div class="commentColumn WikispacesStream"><div class="WikispacesContent WikispacesBs3"><div class="commentList"></div></div></div>
</div>

  




<script type="text/template" id="js-event-template">
  <small>
    <%- startDateShortFormat %>
    <a href="<%- ws.common.pathFor('event', 'view', id) %>" class="WikispacesDialogLink">
      <%- ws.common.truncate(title, 15) %>
    </a>
  </small>
</script>

<script type="text/template" id="js-reply-template">
  <div class="tc-timeline-comment media">

    <a class="pull-left userLink" href="<%- userCreated.url %>">
      <img src="<%- userCreated.imageUrl %>" width="30" height="30" class="author-avatar" alt="" />
    </a>
    <div class="media-body" data-id="<%= id %>">
      <% if (ws.context.user.canAdministerWiki) { %>
        <i class="tc-delete-comment fa fa-trash-o contextual-command destroy-message ws-tip" title="Delete reply"></i>
      <% } %>
      <div class="tc-box-name-date">
        <span class="tc-commenter"><a href="<%- userCreated.url %>"><%- userCreated.username %></a></span>
        <% if (smartDate == '__live__') { %>
        <abbr style="border-bottom-style: none;" class="timeago tc-comment-date" title="<%- (new Date(parseInt(dateCreated) * 1000)).toISOString() %>">
          <%- (new Date(parseInt(dateCreated) * 1000)).toDateString() %>
        </abbr>
        <% } else { %>
          <abbr style="border-bottom-style: none;" class="tc-comment-date"><%- smartDate %></abbr>
        <% } %>
      </div>
      <% if (body) { %>
      <p>
        <%= body %>
      </p>
      <% } %>
    </div>
  </div>
</script>

<script type="text/template" id="js-share-template">
  <div class="tc-timeline-block tc-<%= ws.stream.typeToBlockType(sharedType) %>-block" <%= (ws.stream.hideTimeline ? 'style="padding-left: 0px;"' : '') %>>

    <% if (!ws.stream.hideTimeline) { %>
      <div class="tc-timeline-icon">
        <i class="fa fa-<%= ws.stream.typeToIcon(sharedType) %>"></i>
      </div>
      <div class="tc-timeline-triangle-man"></div>
    <% } %>

    <div class="tc-timeline-box" data-id="<%= id %>">
      <div class="media">
        <a class="pull-left userLink" href="<%- userCreated.url %>">
          <img src="<%- userCreated.imageUrl %>" width="48" height="48" class="author-avatar media-object" alt="" />
        </a>

        <div class="media-body">

          <div class="tc-stream-dropdown">
            <a class="dropdown-toggle" data-toggle="dropdown" href="#">
              <i class="fa fa-chevron-down"></i>
            </a>
            <ul class="dropdown-menu dropdown-menu-right" role="menu">
              <% if (ws.context.user.id) { %>
              <li>
                <a class="js-toggle-monitor" tabindex="-1" href="#">
                  <i class="fa-fw fa fa-eye"></i>
                  <span><%= monitored ? 'Start Monitoring' : 'Stop Monitoring' %></span>
                </a>
              </li>
              <% } %>

              <% if (ws.context.user.canAdministerWiki) { %>
              <li>
                <a class="js-toggle-lock" tabindex="-1" href="#">
                  <i class="fa-fw fa fa-lock"></i>
                  <span><%= locked ? 'Unlock Replies' : 'Lock Replies' %></span>
                </a>
              </li>
              <% if (sharedType == 'discussion' || sharedType == 'comment' || sharedType == 'event') { %>
              <li>
                <a class="js-destroy-share" tabindex="-1" href="#">
                  <i class="fa-fw fa fa-trash-o"></i>
                  Delete <%= ws.stream.typeToName(sharedType) %>
                </a>
              </li>
              <% } %>
              <% } %>
            </ul>
          </div>

          <div class="tc-timeline-box-header ws-reveal media-heading">
              <h3 class="tc-box-title">
              <% if (typeof adminLink !== 'undefined' && adminLink && ws.context.user.canAdministerWiki) { %>
                <a href="<%- adminLink %>"><%- title %></a>
              <% } else { %>
                <%- title %>
              <% } %>
              <% if (typeof startDate !== 'undefined' && startDate) { %> - <%= startDateShortFormat %> <%= ws.stream.getYearFromStartDate(startDate) %><% } %>
              </h3>
              <div class="tc-box-name-date">
                <% if (ws.context.instance.isTesClassroom) { %>
                  <%- userCreated.username %>
                <% } else { %>
                  <a class="author-name" href="<%- userCreated.url %>"><%- userCreated.username %></a>
                <% } %>
                <% if (typeof recipients !== 'undefined' && recipients) { %>
                  <i class="fa fa-angle-right"></i>
                  <% if (typeof recipientsLink !== 'undefined') { %>
                    <a href="<%- recipientsLink %>"><%- recipients %></a>
                  <% } else { %>
                    <span class="recipients"><%- recipients %></span>
                  <% } %>
                <% } %>
                <% if (smartDate == '__live__') { %>
                  <abbr style="border-bottom-style: none;" class="timeago tc-block-date" title="<%= (new Date(parseInt(dateCreated) * 1000)).toISOString() %>">
                    <%= (new Date(parseInt(dateCreated) * 1000)).toDateString() %>
                  </abbr>
                <% } else { %>
                  <abbr style="border-bottom-style: none;" class="tc-block-date"><%- smartDate %></abbr>
                <% } %>
              </div>
          </div>

          <div class="<%= sharedType == 'comment' && !orphaned && !ws.stream.noCollapse ? 'hidden ' : '' %>tc-block-content">
            <div class="clearfix">
              <% if (typeof rangeActions !== 'undefined') { %>
                <div class="tc-range-actions">
                  <% for(var i = 0; i < rangeActions.length; i++) { %>
                  <%= ws.stream.rangeActionTemplate(rangeActions[i]) %>
                  <% } %>
                </div>
              <% } %>
              <% if (description) { %>
                <p><%= description %></p>
              <% } %>
            </div>
          </div>

          <% if (replyPages.length > 0 && !(replyPages.length === 1 && replyPages[0].digests.length === 0)) { %>
            <div class="tc-timeline-comments">
            <% for (var i = 0; i < replyPages.length; i++) { %>
              <% if (i == 0 && replyPages[i].more) { %>
                <a href="#" class="tc-show-more" onclick="ws.stream.pageReplies(<%= id %>, <%= replyPages[i].page + 1 %>); return false">Show more replies (<%= replyPages[i].more %> hidden) </a>
              <% } else { %>
                <div></div>
              <% } %>
              <% for (var j = 0; j < replyPages[i].digests.length; j++) { %>
                <%= ws.stream.replyTemplate(replyPages[i].digests[j]) %>
              <% } %>
            <% } %>
            </div>
          <% } %>

          <% if ((ws.stream.userCanPostMessage && !locked) || ws.context.user.canAdministerWiki) { %>
          <div class="tc-add-comment media <%= ws.stream.noCollapse ? 'add-active' : ''%>">
            <img src="<%- ws.context.user.imageUrlPrefix %>lg.jpg" width="30" height="30" class="pull-left author-avatar" alt="" />
            <div class="media-body">
              <textarea name="reply" placeholder="Comment" class="form-control form-control-tc ws-reveal <%= !ws.stream.noCollapse ? 'ws-unrevealed' : ''%> reply" /></textarea>
              <button type="button" class="btn-tc btn-info-tc btn-sm-tc add-reply">Post</button>
            </div>
          </div>
          <% } %>
        </div>

      </div>
    </div>
  </div>
</script>

<script type="text/template" id="js-range-action-template">
  <div>
    <strong><%- name %></strong>
    <% if (typeof(diffUrl) !== 'undefined') { %>
      <a href="<%- diffUrl %>">
    <% } %>
    <%= new Date(dateCreated * 1000).toLocaleString() %>
    <% if (typeof(diffUrl) !== 'undefined') { %>
      </a>
    <% } %>
    <% if (typeof(rangeText) !== 'undefined') { %>
      <div>
        <span class="commentExtent">
        <% if (typeof(diffUrl) !== 'undefined') { %>
          <a href="<%- diffUrl %>">
        <% } %>
        <%= rangeText %>
        <% if (typeof(diffUrl) !== 'undefined') { %>
          </a>
        <% } %>
        </span>
      </div>
    <% } %>
  </div>
</script>

<script type="text/template" id="js-stream-end-template">
  <div class="tc-timeline-block tc-end-block">
    <div class="tc-timeline-icon" style="margin-top: -10px;" />
    <div class="tc-timeline-box" style="visibility: hidden;">
      <p class="tc-box-title"></p>
    </div>
  </div>
</script>

<a name="discussion"></a>
<div class="timeline WikispacesBs3 tc-timeline-embedded show-border">
  <div class="timeline-line"></div>
    <div id="js-share-list">
    <div class="spinner hidden"><i class="fa fa-spinner fa-spin"></i></div>
  </div>
</div></div></div><div id="sentinel" style="clear:both"></div>
    </div>
  </td>
</tr>
</table>

<div id="foot">
<div class="WikiLicense WikiElement">Contributions to http://mesquiteproject.wikispaces.com/ are licensed under a <a rel="license" href="http://www.creativecommons.org/licenses/by/3.0">Creative Commons Attribution 3.0 License</a>. <a rel="license" href="http://www.creativecommons.org/licenses/by/3.0"><img src="http://www.wikispaces.com/i/creativecommons/by_3.0_80x15.png" width="85" height="15" alt="Creative Commons Attribution 3.0 License"/></a><br />Portions not contributed by visitors are Copyright 2018 Tangient LLC<br /><a href="https://www.tes.com/us/?utm_source=wikispaces&amp;utm_medium=link&amp;utm_campaign=US-52-footer">TES: The largest network of teachers in the world</a></div></div>


<script type="text/template" id="dialog-template">
<div class="modal fade" role="dialog" aria-hidden="true">
  <div class="modal-dialog">
    <div class="modal-content">
      <div class="modal-header">
        <button type="button" class="close" data-dismiss="modal" aria-hidden="true"><i class="fa fa-times"></i></button>
        <h4 class="modal-title"><%= header %></h4>
      </div>
      <div class="modal-body">
        <%= body %>
      </div>
    </div>
  </div>
</div>
</script>


<div id="js-help-template" class="hidden">
  <div class="ws-help">
    <div class="js-help-controls WikispacesContent WikispacesBs3">
      <a href="http://www.wikispaces.com/site/gettingstarted/off" class="btn btn-default hidden js-getting-started-off pull-right">Turn off "Getting Started"</a>
      <ol class="breadcrumb">
        <li class="js-special-breadcrumb active">
          <a href="http://helpcenter.wikispaces.com/" data-breadcrumb="0">Home</a>
        </li>
        <li class="js-ellipsis hidden js-special-breadcrumb">
          ...
        </li>
        <li class="js-spinner hidden js-special-breadcrumb"><i class="fa fa-spinner fa-spin"></i></li>
      </ol>
    </div>
    <div class="ws-help-content wiki">Loading...</div>
  </div>
</div>
</div>
<script src="http://www.wikispaces.com/s/gz/j4257158-common.min.js" type="text/javascript"></script>
<script src="http://www.wikispaces.com/s/gz/85k7018k-stream.min.js" type="text/javascript"></script>
<script src="http://www.wikispaces.com/s/gz/91k8z207-contentbar.min.js" type="text/javascript"></script>
<script src="http://www.wikispaces.com/s/gz/395j1vq7-component.stream-vo.min.js" type="text/javascript"></script>
<script type="text/javascript">
<!--
ws.namespace('ws.attributes');
ws.namespace('ws.stream');
ws.attributes.DISCUSSION_COUNT = 0;
ws.attributes.pageId = 200870446;
ws.attributes.version = 586165417;
ws.attributes.showEditor = false;
ws.attributes.editUrl = "Analyzing%2bMolecular%2bData06c1.html\/\/mesquiteproject.wikispaces.com\/page\/edit\/Analyzing+Molecular+Data?goto=http%3A%2F%2Fmesquiteproject.wikispaces.com%2FAnalyzing%2BMolecular%2BData";
ws.attributes.userCanEdit = false;
ws.stream.userCanPostMessage = false;
ws.stream.hideTimeline = true;
ws.stream.shareSource = "Analyzing%2bMolecular%2bData06c1.html\/\/mesquiteproject.wikispaces.com\/page\/shares\/Analyzing+Molecular+Data";

  (function() {
    var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
    ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
  })();

jQuery(document).ready(function() {
  ws.common.addJsonNotices({"warning":[],"error":[],"info":[],"success":[]});
});

jQuery(document).ready(function() {
  ws.editor.canEdit = false;
  var editUrl = 'http://mesquiteproject.wikispaces.com/page/edit/Analyzing+Molecular+Data?goto=http%3A%2F%2Fmesquiteproject.wikispaces.com%2FAnalyzing%2BMolecular%2BData';

  jQuery('#js-edit-from-template').change(function() {
    window.location = ws.common.addUriParams(editUrl, [['template', jQuery(this).val()]]);
  });
});



--></script><!-- w24 94 tot:94:94(1) mem:3:11(18) wki:1:1(1) -->

</body>

<!-- Mirrored from mesquiteproject.wikispaces.com/Analyzing+Molecular+Data?f=print by HTTrack Website Copier/3.x [XR&CO'2014], Tue, 01 May 2018 13:27:42 GMT -->
</html>